samples/batch_parse_form.js

```diff
@@ -29,7 +29,7 @@ async function main(
    * TODO(developer): Uncomment these variables before running the sample.
    */
   // const projectId = 'YOUR_PROJECT_ID';
-  // const location = 'YOUR_PROJECT_LOCATION',
+  // const location = 'Project location--Must be in format \'us\' or \'eu\'';
```

Comment:
You shouldn't need the `\` on this, I might just say something like:

```
'YOUR_PROJECT_LOCATION' // format "us", or "eu".
```

☝️ looks a bit cleaner, and still works when you uncomment.

Reply:
Done.